### PR TITLE
Add click tp

### PR DESCRIPTION
**Pull request by koliwbr**
## Description
Added click Tp

## (Optional) screenshots / videos
Type `.tp` to teleport to the indicated block

### CODE REVIEW

File: src/main/java/net/wurstclient/commands/TpCmd.java [hunk 1]
```diff
@@ -34,7 +41,7 @@ public void call(String[] args) throws CmdException
 		BlockPos pos = argsToPos(args);
 		
 		ClientPlayerEntity player = MC.player;
-		player.setPosition(pos.getX() + 0.5, pos.getY(), pos.getZ() + 0.5);
+		player.setPosition(pos.getX() + 0.5, pos.getY()+0.0, pos.getZ() + 0.5);
```
Comment (InfinitePower563) [inline src/main/java/net/wurstclient/commands/TpCmd.java:44]: What does adding 0.0 do? Just curious.

**Review by InfinitePower563 — APPROVED** [review]
Good addition, however, I see a lot of clients have a ClickTP module. It highlights the selected block, and, on right click, it teleports you to the block. Maybe you could incorporate the command into a module, or you can create a separate PR with this functionality.

File: src/main/java/net/wurstclient/commands/TpCmd.java (hunk 1 above)
Comment (koliwbr) [reply src/main/java/net/wurstclient/commands/TpCmd.java:44]: Idk, maybe trash from debug

Comment (InfinitePower563) [reply src/main/java/net/wurstclient/commands/TpCmd.java:44]: Have you tried removing it and seeing if it works without the addition? I think extra redundant code makes it harder to understand.

**Review by InfinitePower563 — APPROVED** [review]
Nice addition, would love to see a module of ClickTP in the future.

**Review by FlorianMichael — APPROVED** [review]

**Pull request closed without merging**